apps/explorer/lib/blocks.ts

```diff
@@ -1,24 +1,47 @@
 import { to } from '@siafoundation/request'
-import { siaCentral } from '../config/siaCentral'
-import { SiaCentralBlock } from '@siafoundation/sia-central-types'
-import { range } from '@technically/lodash'
-
-export async function getLastFewBlocks(
-  block?: SiaCentralBlock
-): Promise<[SiaCentralBlock[], undefined] | [undefined, Error]> {
-  if (!block) {
-    return [undefined, Error('no block')]
-  }
-  const lastBlockHeight = block.height || 0
-  const [blocks] = await to(
-    siaCentral.blocks({
-      data: {
-        heights: range(lastBlockHeight - 4, lastBlockHeight),
-      },
-    })
+import { ExplorerBlock } from '@siafoundation/explored-types'
+import { explored } from '../config/explored'
+
+export async function getBlockByHeight(height: number) {
+  // Grab the tip at this height.
+  const [tip, tipError] = await to(
+    explored.consensusTipByHeight({ params: { height } })
   )
-  if (blocks?.blocks) {
-    return [[block, ...blocks.blocks], undefined]
+
+  if (tipError) throw tipError
+  if (!tip) return null
+
+  // Grab the block with the ID at this tip height.
+  const [block, blockError] = await to(
+    explored.blockByID({ params: { id: tip.id } })
+  )
+
+  if (blockError) throw blockError
+  if (!block) return null
+
+  return block
+}
+
+export async function getLatestBlocks(
+  numberOfBlocks: number = 5
+): Promise<[ExplorerBlock[], undefined] | [undefined, Error]> {
+  // Grab the latest tip.
+  const [latestTip, latestTipError] = await to(explored.consensusTip())
+  if (latestTipError) throw latestTipError
+  if (!latestTip) return [undefined, Error('no tip')]
+
+  const fetchedBlocks: ExplorerBlock[] = []
+
+  // Fetch the last 5 blocks.
+  for (let i = 1; i <= numberOfBlocks; i++) {
+    const currentBlockHeight = latestTip.height - i
+    const foundBlock = await getBlockByHeight(currentBlockHeight)
```

Comment:
You only need to get the latest block by height. You can use `block.parentID` to get the previous block

Reply:
We should probably just add an endpoint that returns the last `n` blocks though.

Reply:
I didn't realize that's what parentID referred to but that makes sense! That would reduce the necessary requests. Thank you.

As to the new route idea: This replaces a Sia Central request that allows you to specify the block heights:

```
    siaCentral.blocks({
      data: {
        heights: range(lastBlockHeight - 4, lastBlockHeight),
      }
```

Something like this would have more utility and could be used for a last x request with the tip in hand.